packages/volto-iframe-block/src/helpers/isValidUrl.js

```diff
@@ -17,12 +17,12 @@ export const toastError = defineMessages({
   },
 });
 
-export function isValidUrl(url) {
+export function isValidUrl(props) {
```

Comment:
We can't change the type of the first parameter or what it means (and turning it into a props object doesn't match what you are passing from the Edit component).
```suggestion
export function isValidUrl(url, siteData) {
```